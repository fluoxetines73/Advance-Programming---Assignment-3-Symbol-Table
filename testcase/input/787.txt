INSERT c number
ASSIGN c 456 random
END
BEGIN
BEGIN
INSERT e_123_2r string
END
END
PRINT
UWAAAA
BEGIN
RPRINT
END
PRINT
INSERT g_1 number
INSERT b number
BEGIN
ASSIGN g_1 1001
LOOKUP z
RPRINT
LOOKUP b