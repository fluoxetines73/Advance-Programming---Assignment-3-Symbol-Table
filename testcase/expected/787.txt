Invalid: ASSIGN c 456 random